src/source/PeerConnection/Rtp.h

```diff
@@ -10,16 +10,18 @@ extern "C" {
 // Default MTU comes from libwebrtc
 // https://groups.google.com/forum/#!topic/discuss-webrtc/gH5ysR3SoZI
 #define DEFAULT_MTU_SIZE_BYTES                     1200
-#define DEFAULT_ROLLING_BUFFER_DURATION_IN_SECONDS 3
-#define DEFAULT_EXPECTED_VIDEO_BIT_RATE            (5 * 1024 * 1024)
+#define DEFAULT_ROLLING_BUFFER_DURATION_IN_SECONDS (DOUBLE) 3
+#define DEFAULT_EXPECTED_VIDEO_BIT_RATE            (DOUBLE)(5 * 1024 * 1024)
 // Opus has highest based on the supported codecs which is 510Kibps. So setting this to twice
-#define DEFAULT_EXPECTED_AUDIO_BIT_RATE        (1000 * 1024)
+#define DEFAULT_EXPECTED_AUDIO_BIT_RATE        (DOUBLE)(1000 * 1024)
 #define DEFAULT_SEQ_NUM_BUFFER_SIZE            1000
 #define DEFAULT_VALID_INDEX_BUFFER_SIZE        1000
 #define DEFAULT_PEER_FRAME_BUFFER_SIZE         (5 * 1024)
 #define SRTP_AUTH_TAG_OVERHEAD                 10
 #define MIN_ROLLING_BUFFER_DURATION_IN_SECONDS (DOUBLE) 0.1
 #define MIN_EXPECTED_BIT_RATE                  (DOUBLE)(102.4 * 1024) // Considering 1Kib = 1024 bits
+#define MAX_ROLLING_BUFFER_DURATION_IN_SECONDS (DOUBLE) 10
```

Comment:
How were these determined?

Reply:
Hmm. No specific reason. The RTP packets are stored to address nacks and for the jitter buffer. Typically, a buffer of 1-3 seconds should be a decent value for storing packets for retransmissions. Storing 10 seconds worth of RTP packets and retransmitting a 10 second old packet is bad enough for the latency of video streaming and playback. So in my mind, it is pointless to store this. The value is randomly high (set to 10) to have an allowed range of values.